chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/surveillance/ActivityReportActionPermission.java

```diff
@@ -0,0 +1,20 @@
+package gov.healthit.chpl.permissions.domains.surveillance;
+
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.permissions.domains.ActionPermissions;
+
+@Component("surveillanceActivityReportActionPermission")
+public class ActivityReportActionPermission extends ActionPermissions {
+
+    @Override
+    public boolean hasAccess() {
+       return getResourcePermissions().isUserRoleAdmin() || getResourcePermissions().isUserRoleOnc();
```

Comment:
I thought we said, at the FR, that this job would be available for ROLE_ONC_STAFF, even if the UI doesn't give them a way to run it yet. This looks like it's not available